api/src/repositories/occurrence-repository.ts

```diff
@@ -38,6 +45,63 @@ export interface IPostOccurrenceData {
  * @extends {BaseRepository}
  */
 export class OccurrenceRepository extends BaseRepository {
+  /**
+   * Get all occurrences data within mapView
+   *
+   * @return {*}  {Promise<IGetOccurrenceData[]>}
+   * @memberof OccurrenceRepository
+   */
+  async getMapOccurrences(mapView?: string | undefined): Promise<IGetOccurrenceData[]> {
+    const sqlStatement = SQL`
+      SELECT
+        o.occurrence_id,
```

Comment:
super nit: if we only have 1 table in this query, we might as well drop the `as o` and `o.` prefix,just to simplify it slightly.